perfmetrics/scripts/utils/metrics_util_test.py

```diff
@@ -0,0 +1,33 @@
+"""Test for fio output logging."""
+
+import unittest
+import os
+from metrics_util import remove_old_log_files
+
+
+class TestMetricsUtil(unittest.TestCase):
+
+  def test_remove_old_log_files(self):
```

Comment:
Please add test cases for these scenarios
dir contains 5 files and we pass 6
dir contains 0 files and we pass >1